src/formio/components/Fieldset.stories.mdx

```diff
@@ -0,0 +1,146 @@
+import {Canvas, Meta, Story, ArgsTable} from '@storybook/addon-docs';
+import {FormioComponent, FormDecorator} from './story-util';
+
+<Meta
+  title="Form.io components/Fieldset"
+  argTypes={{
+    components: {
+      type: {name: 'array'},
+    },
+    id: {
+      type: {name: 'string'},
+    },
+    label: {
+      type: {name: 'string', required: false},
+    },
+    descriptionFamilyName: {
+      type: {name: 'string'},
+    },
+    descriptionGivenName: {
+      type: {name: 'string'},
+    },
+    labelFirstName: {
+      type: {name: 'string'},
+    },
+    labelLastName: {
+      type: {name: 'string'},
+    },
+    labelPrivacy: {
+      type: {name: 'string'},
+    },
+    placeholderFirstName: {
+      type: {name: 'string'},
+    },
+    placeholderLastName: {
+      type: {name: 'string'},
+    },
+  }}
+  decorators={[FormDecorator]}
+  parameters={{
+    controls: {sort: 'requiredFirst'},
+  }}
+/>
+
+# Field Set
+
+A Field Set can be used to create a title of an area of the form. This is useful to put inside
+Layout components or in between lots of related components. The Field Set is for display only and
+will not be saved to the API,
+
+Additionally the use of a field set increases accessibility: grouping related form controls makes forms more understandable for all users, as related controls are easier to identify, makign it easier for people to focus on smaller and more manageable groups.
+
+**Legend**
+
+A Field Set label that will appear as a Legend for the component in the form.
+
+## Default
+
+export const Template = ({ descriptionFamilyName, descriptionGivenName, id = '', label = '', labelFirstName, labelLastName, labelPrivacy, placeholderFirstName, placeholderLastName, ...args }) => (
```

Comment:
In this case I would structure the args quite differently - there are a number of args/controls directly related to the fieldset:

* `label`: content of the fieldset legend
* `id` -> mapped to formio `key`: `fieldset` as default value is fine
* `hideHeader`: controls whether the label is visible or not

And then there's the nested components inside the fieldset - you have a number of args that related to these nested components, but they now prevent me from pasting completely different components into the configuration in storybook to see how that looks. What if I wanted to check a `select` component?

So it'd be better to provide an arg `nestedComponents: [...]` with the current nested components, and leave it so you can edit that as one big blob.